### ISSUE TITLE
Booking - special requests

### ISSUE THREAD

**Issue by stringiest** [stringiest/new-callacon#15]
Non cost-related options:
- lark or owl
- massage house
- late & loud
- chill-out house
- family room
- Who are you sharing with? If you are sharing, would you prefer a double bed (or two singles)? 
- hot tub? 
- Would you prefer to be in a house that has a dog?
- add general booking comments box for user

**Comment by hopshackle** [comment]
Yes - not necessarily suggesting this for MVP - but this information is really 'static' and could be stored as part of a person's profile rather than separately on every single booking?

**Comment by stringiest** [comment]
Yes, could extract this for a future version. Maybe we display the person's previous choices, and they can amend if necessary.
